app.rb

```diff
@@ -1,8 +1,20 @@
 require 'sinatra/base'
+require './lib/peeps'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
+  # get '/' do
```

Comment:
I would probably try to avoid leaving commented out code in your code base, either you need that redirect or you don't